pal/samples/esp8266/iothub_ll_telemetry_sample.ino

```diff
@@ -220,5 +240,18 @@ void setup() {
 
 void loop(void)
 {
+    if (Serial.available()){
+        String s1 = Serial.readStringUntil('\n');// s1 is String type variable.
+        Serial.print("Received Data: ");
+        Serial.println(s1);//display same received Data back in serial monitor.
   
+        int e_start = s1.indexOf('e');
+        String ebit = (String) s1.substring(e_start, e_start+4);
+        if(ebit == "exit")
+        {
+#ifdef is_esp_board
```

Comment:
what use is this code if `is_esp_board` is not defined? this loop and the above code in the setup don't seem to do much.

Reply:
I guess I could just define around the whole thing. I kept the defines narrow so as to indicate that the defines are just there for ESP specific code.